### DISCUSSION TITLE
config.ts does't hot reload when the  navbar config is change

### DISCUSSION THREAD

**Discussion by RenshuozZ** [vuepress/core#235] [Q&A] [answered]
config.ts does't hot reload when the  navbar config is change

**Comment by yzx9** [comment]
Please provide a repro of your question 😊

**Reply by Mister-Hope** [reply]
The official docs could be a repro.

**Reply by Mister-Hope** [reply]
I am finding out sidebars not changed under some situation too. There could be some issues.

**Comment by meteorlxy** [comment]
Modifying config file will restart the dev-server.

**Comment by Mister-Hope** [comment, marked as answer]
This is probably  fixed in latest version, could you check it again in `bete.35`?
